core-web/libs/portlets/dot-ema/src/lib/feature/dot-ema.component.scss

```diff
@@ -5,6 +5,18 @@
     flex-direction: column;
     grid-template-rows: 48px 1fr;
     height: 100%;
+
+    ::ng-deep {
+        .p-dialog {
+            .p-dialog-content {
```

Comment:
You can use the `contentStyleClass` [Dialog Input](https://primeng.org/dialog#api.dialog.props.contentStyleClass)

Reply:
Ohhhh that's better.

Reply:
I tried to do that, but for some reason the styles does not apply